### PR TITLE
[10.0][UPD] Add the invoice/get service

### PR DESCRIPTION
**Pull request by acsonefho**
Add the invoice/get service (+ related unit test).

Also fix existing unit test (to check json result (`_check_data_content`), who doesn't use the given `invoices` var).

Fail due to missing MR (sale-workflow). Fixed into #505

### CODE REVIEW

File: shopinvader_invoice/tests/test_invoice_service.py [hunk 1]
```diff
@@ -167,3 +167,25 @@ def test_get_multi_invoice(self):
         data = result.get("data", [])
         self._check_data_content(data, invoices)
         return
+
```
Comment (simahawk) [inline shopinvader_invoice/tests/test_invoice_service.py:171]: I don't see any test for not being able to fetch invoices that belong to other partners. Correct?

Comment (acsonefho) [reply shopinvader_invoice/tests/test_invoice_service.py:171]: You're right, I don't write this kind of test for this service

**Comment by Cedric-Pigeon** [comment]
@simahawk is this one ok for you ?

**Review by Cedric-Pigeon — APPROVED** [review]

**Review by simahawk — APPROVED** [review]

**Pull request merged**